ocamlorg.opam

```diff
@@ -25,7 +25,7 @@ depends: [
   "bos"
   "crunch"
   "mirage-kv-mem"
-  "dream" {>= "1.0.0~alpha5"}
+  "dream" {>= "1.0.0~alpha3"}
```

Comment:
Why is dream listed twice - and with two different versions? :thinking:

Reply:
It shouldn't be duplicated. I was in main yesterday. I will rebase this

Reply:
Thanks!